### PR TITLE
S: Added a directory for engineering leadership, and passed proposals

### PR DESCRIPTION
**Pull request by maxamus123**
* added readme with expected format for recording passed proposals
* created file for passed proposals of December 2022
* updated box_of_autonomy.md to reflect passed proposal

### CODE REVIEW

File: box_of_autonomy.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Each team/product will produce/publish/follow the following:
 
 ## Development Time Box
 - Maximum delivery to production time box of 1 week
+- Sprint lengths of 1 week
```
Comment (dneighbors) [inline box_of_autonomy.md:27]: - Maximum delivery to production time box of 1 week
was originally a way to enforce regular delivery without constraining to a particular iteration length or style.. we probably should remove it in favor of what was approved/added

File: engineering_leadership/passed_proposals/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+This is a place to keep track of passed proposals made through engineering leadership channels.
+## Format of Proposals
+***
+#### Date
+12-06-22
+#### Proposal
```
Comment (dneighbors) [inline engineering_leadership/passed_proposals/readme.md:6]: should this include a link to the proposal in slack (if it was made digitally)?

File: engineering_leadership/passed_proposals/12_2022.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+***
+#### Date
+12-06-22
+#### Proposal
+I propose that all engineering teams move to one-week sprints  1, 2, 3
```
Comment (dneighbors) [inline engineering_leadership/passed_proposals/12_2022.md:5]: should this include a link to the proposal in slack (if it was made digitally)?

**Review by dneighbors — CHANGES_REQUESTED** [review]

File: box_of_autonomy.md (hunk 1 above)
Comment (maxamus123) [reply box_of_autonomy.md:26]: okay, awesome, I removed it.

File: engineering_leadership/passed_proposals/readme.md (hunk 2 above)
Comment (maxamus123) [reply engineering_leadership/passed_proposals/readme.md:6]: it absolutely can, done

**Review by dneighbors — APPROVED** [review]

**Pull request merged**